src/Nox.Solution/Validation/Domain/EntityValidator.cs

```diff
@@ -71,6 +71,12 @@ public EntityValidator(IEnumerable<Entity>? entities, Application? application)
 
             RuleForEach(p => p.OwnedRelationships)
                .SetValidator(x => new EntityItemUniquenessValidator<EntityRelationship>(x, t => t.Name, nameof(x.OwnedRelationships)));
+
+            RuleForEach(e => e.UniqueAttributeConstraints)
+                .SetValidator(v => new UniqueAttributeConstraintValidator(v))
+                .SetValidator(e => new UniquePropertyValidator<UniqueAttributeConstraint>(e.UniqueAttributeConstraints, x => x.Name, "unique attribute constraint"))
+                .SetValidator(e => new UniquePropertyValidator<UniqueAttributeConstraint>(e.UniqueAttributeConstraints, x => string.Join(",", x.AttributeNames.OrderBy(a => a)), "unique attribute constraint attribute names"));
```

Comment:
Should this validation be withing the UniqueAttributeConstraintValidator, since it validates a property of the uniqueAttributeConstraint?